### ISSUE TITLE
Add DISABLED state for buttons, inputs and so on

### ISSUE THREAD

**Issue by matzimowski** [brainly/style-guide#190]

**Comment by vergilius** [comment]
@matzimowski is working on buttons right now

**Comment by matzimowski** [comment]
done for buttons

**Comment by clxandstuff** [comment]
Done

**Comment by clxandstuff** [comment]
Reopen, input still missing disabled state
